examples/create_user.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE User (
+  id INT NOT NULL AUTO_INCREMENT,
```

Comment:
/ask explain this line

Reply:
This line defines a column named `id` in the `User` table with several attributes:

- `INT`: Specifies the data type of the column as an integer, meaning it can store whole numbers.
- `NOT NULL`: This constraint ensures that the column cannot have a NULL value, meaning every user must have an ID.
- `AUTO_INCREMENT`: This attribute is used for automatically incrementing the value of the `id` column each time a new record (user) is inserted into the table. It helps in creating a unique identifier for each user without manually specifying the ID.